src/translations/sqlb_es_ES.ts

```diff
@@ -2333,7 +2333,7 @@ Do you want to insert it anyway?</source>
     </message>
     <message>
         <location filename="../MainWindow.ui" line="1891"/>
-        <source>Copy the CREATE statement of the item to the clipboard</source>
+        <source>Copy the CREATE statement of the ítem to the clipboard</source>
```

Comment:
Not sure here because this changes the source string (= the original English text), not the Spanish one.